services/docker-compose-ops.yml

```diff
@@ -69,6 +69,24 @@ services:
       timeout: 20s
       retries: 3
 
+  filestash:
+    container_name: filestash
+    image: machines/filestash:3a01b70
+    restart: always
+    ports:
+      - "9002:8334"
```

Comment:
is that a good port? I think minio 1-4 run on 9000-9003.

Reply:
This is the local published port. Usually minio is exposed on 9001. Not sure if this will collide with something else.